go/pkg/yolostore/gorm.go

```diff
@@ -0,0 +1,414 @@
+package yolostore
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"strings"
+
+	"berty.tech/yolo/v2/go/pkg/yolopb"
+	"github.com/jinzhu/gorm"
+	"github.com/mr-tron/base58"
+	"go.uber.org/zap"
+	"moul.io/zapgorm"
+)
+
+type Store interface {
```

Comment:
can you move the store and its methods to a `store.go` file and keep in `gorm.go` everything only related to gorm (gorm init, generic helpers, etc)